lib/utils.js

```diff
@@ -149,4 +149,17 @@ function insertAppiumPrefixes (caps) {
   return prefixedCaps;
 }
 
-export { inspectObject, parseCapsForInnerDriver, insertAppiumPrefixes };
+/**
+ * Get package.json object
+ */
+function getPkgObject () {
```

Comment:
this is duplicated in several places, why not make it part of something more global?